### DISCUSSION TITLE
Authenticate Cloud Run

### DISCUSSION THREAD

**Discussion by thalyswolf** [stackkit/laravel-google-cloud-tasks-queue#50] [General] [answered]
Hi there  👋🏻,

I have a problem and I hope someone can help me.

I`m using Laravel 8 (PHP 8) and Cloud Run. I get to create an Job but I don't get to handle this Job because it occurs an erro in authentication, follow this log:

``` console
UnexpectedValueException: Audience does not match in /app/vendor/google/auth/src/AccessToken.php:271
```

Someone would known if is need more some step for working in cloud run.

Thank you in advance

**Comment by marickvantuil** [comment, marked as answer]
Hi, this was a bug in 3.x which I just released a patch for: 3.1.1

Can you try with that version and let me know if it works?

**Reply by thalyswolf** [reply]
Yes, it worked.

Thank u very much! 😀
